Src/FluentAssertions.Json/FluentAssertions.Json.csproj

```diff
@@ -12,7 +12,7 @@
     <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="FluentAssertions" Version="5.0.0" />
+    <PackageReference Include="FluentAssertions" Version="5.10.2" />
```

Comment:
I'm fine with the upgrade, but out of curiosity, is upgrading the core package part of the fix?

Reply:
We're using the newer version of FluentAssertions in our project, so we see the wrong error messages. Apparently the problem is hidden by FA in the older version.